proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -2282,6 +2437,232 @@
+            "$value": "{basis.color.info.text-1}"
+          }
+        },
+        "error": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.error.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.error.text-1}"
+          }
+        },
+        "ok": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.success.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.success.text-1}"
+          }
+        },
+        "warning": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.bg-1}"
+          },
+          "border-color": {
+            "$type": "color",
+            "$value": "transparent"
+          },
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.warning.text-1}"
+          }
+        }
+      }
+    }
+  },
+  "components/select": {
+    "utrecht": {
+      "select": {
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "border-radius": {
+          "$type": "dimension",
+          "$value": "{basis.border-radius.md}"
+        },
+        "focus": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
+          }
+        },
+        "invalid": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
+          }
+        }
+      }
+    }
+  },
+  "components/textarea": {
+    "utrecht": {
+      "textarea": {
+        "background-color": {
+          "$type": "color",
+          "$value": "{basis.color.text.interactive-1}"
+        },
+        "border-radius": {
+          "$type": "dimension",
+          "$value": "{basis.border-radius.md}"
+        },
+        "padding-block-start": {
+          "$type": "dimension",
+          "$value": "{basis.space.block.x-large}"
+        },
+        "padding-block-end": {
+          "$type": "dimension",
+          "$value": "{basis.space.block.x-large}"
+        },
+        "placeholder": {
+          "color": {
+            "$type": "color",
+            "$value": "{basis.color.disabled.text-2}"
+          }
+        },
+        "focus": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
+          }
+        },
+        "invalid": {
+          "background-color": {
+            "$type": "color",
+            "$value": "{basis.color.text.interactive-1}"
```

Comment:
`invalid` is toch geen interactive state? Het lijkt me dat deze een gewone achtergrondkleur moet hebben. In het basis thema is de waarde `"$value": "{basis.form-control.background-color}"`. Is het echt nodig hier vanaf te wijken voor Purmerend?

Nog een probleem met interactive gebruiken: dan heb je ook geen verschil meer als een invalid form control `hover` state heeft.